plugin/src/main/kotlin/me/gabber235/typewriter/entry/entries/EventEntry.kt

```diff
@@ -1,15 +1,31 @@
 package me.gabber235.typewriter.entry.entries
 
+import me.gabber235.typewriter.Typewriter.Companion.plugin
 import me.gabber235.typewriter.adapters.Tags
 import me.gabber235.typewriter.entry.Entry
 import me.gabber235.typewriter.entry.TriggerEntry
+import me.gabber235.typewriter.entry.triggerAllFor
+import org.bukkit.command.CommandSender
+import org.bukkit.command.defaults.BukkitCommand
 import org.bukkit.entity.Player
 
 @Tags("event")
-interface EventEntry : TriggerEntry {
-	// make a command val that does not need to be overridden
+interface EventEntry : TriggerEntry
+
+interface CustomCommandEntry : EventEntry {
 	val command: String
-		get() = ""
+
+	fun register() {
+		val result = plugin.getCommandMap().register(command, object : BukkitCommand(command) {
+			override fun execute(sender: CommandSender, commandLabel: String, args: Array<out String>): Boolean {
+				triggerAllFor(sender as Player)
+				println("Triggered command $command for $name (${id})")
+				return true
+			}
+		})
+
+		println("Registered command $command for $name (${id}) Success: $result")
+	}
```

Comment:
Here is the part I am not sure about. Whenever I start the server, it works just fine and registers the commands as expected. As soon as I push new changes though, it does not register any commands, and when triggered the `execute` function does nothing.

Reply:
From what I can remember, is that the changes to the command map need to be synced again. I dont remember if it was with the server or client. To discover that, try `starting server -> joining -> pushing changes -> testing (shoudn't work) -> leave server -> join server -> test again (t2)`. If the second test `(t2)` works than we need to sync it with the client other wise the server.

Reply:
Sounds like the server then, I tested it and it did not work.

Reply:
Have a look at this, https://www.spigotmc.org/threads/cannot-register-command-with-commandmap.577072/ I think this is the way to solve it